samples/apps/copilot-chat-app/webapi/Config/ServiceOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Wouldn't the service port fit in here?

Reply:
ASP.NET already has a mechanism (several, actually) for defining how to bring up the endpoints. I chose to use the Kestral endpoints appsettings method.